solidity/dashboard/src/utils/ethereum.utils.js

```diff
@@ -46,14 +47,25 @@ export const getEventsFromTransaction = async (contractToEventName, txHash) => {
   return events
 }
 
+// We want to prevent long values being displayed in ETH unit(eg.
+// 0.000034560123345621 ETH). So in that case we want to display `<0.0001`.
+// More info here:
+// https://github.com/keep-network/keep-core/pull/2050#issuecomment-693434991
+export const MIN_ETH_AMOUNT_TO_DISPLAY_IN_WEI = "100000000000000" // 0.0001 ETH
```

Comment:
This might be easier to understand if it was something like `toBN(10).pown(13)`.

Reply:
pls see 6610f87